meson.build

```diff
@@ -51,6 +51,15 @@ if get_option('with_encode2')
   sources += files('encode2.c')
 endif
 
+if cc.get_argument_syntax() == 'msvc'
+  add_project_arguments('-D_CRT_SECURE_NO_WARNINGS',
+                        '/wd4244',
+                        '/wd4245',
+                        '/wd4267',
+                        '/wd4310',
```

Comment:
Could you add a comment for each of these options to describe what they do -- for people who don't regularly use MSVC?